components/mpas-ocean/src/analysis_members/Registry_time_series_stats_monthly_mean.xml

```diff
@@ -193,11 +208,28 @@
 			<var name="SSHSquared"/>
 			<var name="velocityZonalSquared"/>
 			<var name="velocityMeridionalSquared"/>
+			<var name="normalVelocitySquared"/>
 			<var name="velocityZonalTimesTemperature"/>
 			<var name="velocityMeridionalTimesTemperature"/>
-                        <var name="velocityZonalTimesTemperature_GM"/>
-                        <var name="velocityMeridionalTimesTemperature_GM"/>
-                        <var name="GMBolusVelocityZonal"/>
-                        <var name="GMBolusVelocityMeridional"/>
+			<var name="normalVelocityTimesTemperature"/>
+			<var name="oceanHeatContentSfcToBot"/>
+			<var name="oceanHeatContentSfcTo700m"/>
+			<var name="oceanHeatContent700mTo2000m"/>
+			<var name="oceanHeatContent2000mToBot"/>
+			<var name="activeTracerForcingMLTend"/>
+			<var name="activeTracerHorMixMLTend"/>
+			<var name="activeTracerNonLocalMLTend"/>
+			<var name="activeTracerVertAdvectionMLTend"/>
+			<var name="activeTracerHorAdvectionMLTend"/>
+			<var name="activeTracerVertMixMLTend"/>
+			<var name="activeTracersML"/>
```

Comment:
I think all these `activeTracer*` requests need to be `var_array name` -- similar to the full 3d activeTracer tends above and below

Reply:
@darincomeau, we really need to make an issue (and eventually a PR) to fix these in E3SM.

Reply:
@vanroekel, this might explain the error I'm seeing (rather than the registry order).  But it seems safest to fix both.

Reply:
@xylar I definitely agree with you.  I saw your comment on the other fix right after posting this comment

Reply:
Sorry I missed this. Fix has been pushed, compiles for me and passed compass's performance test.